src/pages/HomePage.tsx

```diff
@@ -10,9 +16,45 @@ import { WhiteoutAkatsukiLogo } from "../components/images/logos/WhiteoutAkatsuk
 
 export const HomePage = () => {
   // TODO: fetch these from a backend API
-  const totalPPEarned = 1_483_238
-  const totalScoresSet = 92_383_238
-  const totalUsersRegistered = 172_395
+  const [totalPPEarned, setTotalPPEarned] = useState(0)
+  const [totalScoresSet, setTotalScoresSet] = useState(0)
+  const [totalUsersRegistered, setTotalUsersRegistered] = useState(0)
```

Comment:
might want to hide the uis in this null state

Reply:
using `0` as a default state rather than `null` or `undefined` smells weird to me...